symphony-bdk-spring/symphony-bdk-app-spring-boot-starter/src/main/java/com/symphony/bdk/app/spring/properties/CorsProperties.java

```diff
@@ -18,10 +19,17 @@
    */
   private List<String> allowedOrigins = Collections.singletonList("/**");
 
+  /**
+   * [deprecated] use property "allow-credentials" instead
```

Comment:
I think we can also use @deprecated in the Javadoc

Reply:
my bad, just updated